source/dubregistry/mirror.d

```diff
@@ -53,21 +53,31 @@ nothrow {
 	try {
 		DbPackage[] packs;
 		URL url;
-		auto path = NativePath(fileOrUrl);
+
+		const path = NativePath(fileOrUrl);
 
 		string source_text;
 
-		if (path.existsFile) source_text = path.readFileUTF8;
+		if (path.existsFile) {
+			logInfo("Using local file at %s", path);
+			source_text = path.readFileUTF8;
+		}
 		else {
 			url = URL(fileOrUrl);
+
+			const uri = url ~ InetPath("api/packages/dump");
+
+			logInfo("Downloading packages from %s", uri);
+
```

Comment:
- A bit less empty lines would be easier to read. Having one empty line between each statement is wasteful;
- `uri` can be re-used in `requestHTTP`

Reply:
Fixed